readme.md

```diff
@@ -8,28 +8,46 @@
 
 ## Overview
 
+Chief-of-state (COS) is a dockerized persistence framework for building modern, clustered, reactive applications.
+COS encourages best practices in CQRS and event-sourcing through simple, language-agnostic interfaces via gRPC, and it
+allows developers to describe their schema with Protobuf. Under the hood, COS leverages Akka to scale out and guarantee
+performant, reliable persistence.
+
+Chief-of-state was built by Namely with the following principles:
+* Wire format should be the same as persistence (Protobuf is a better schema definition than DDL)
+* Prototypes should scale
+* Developers should never face race conditions or database locks
+* A perfect datastore would offer random access by key, streaming semantics, and atomic writes
```

Comment:
I like datastores that support complex querying. Perhaps:

A ~perfect~ datastore ~would offer~ that offers random access by key, streaming semantics, and atomic writes is a good match for an event sourced application.